ob-chatgpt-shell.el

```diff
@@ -104,14 +124,19 @@ This function is called by `org-babel-execute-src-block'"
                                        '((chatgpt-shell . t))))
   (add-to-list 'org-src-lang-modes '("chatgpt-shell" . text)))
 
-(defun ob-chatgpt--relevant-source-blocks-before-current ()
-  "Return all previous source blocks from current one."
+(defun ob-chatgpt--relevant-source-blocks-before-current (context-name)
+  "Return all previous source blocks relative to the current block with a
+:context arg with a value matching `CONTEXT-NAME'. If
```

Comment:
nit: No need to quote it. Uppercase should be enough.

`M-x checkdoc` seems to accept both, but may as well remain consistent within same file.